partcad/src/partcad/test/cam_forming.py

```
@@ -0,0 +1,44 @@
+from ..part import Part
+from ..part_config import PartConfiguration
+from ..part_config_manufacturing import METHOD_FORMING
+
+from OCP.ShapeAnalysis import ShapeAnalysis_FreeBoundsProperties
+
+
+class CamFormingTest(Test):
+    def __init__(self) -> None:
+        super().__init__("cam-forming")
+
+    async def test(self, tests_to_run: list[Test], ctx, shape, test_ctx: dict = {}) -> bool:
```

Ruff (0.8.2)

Unused method argument: `tests_to_run`

(ARG002)